src/src/DataFixtures/AppFixtures.php

```diff
@@ -11,5 +13,7 @@ class AppFixtures extends Fixture
     public function load(ObjectManager $manager): void
     {
         UserFactory::createMany(10);
+        TagFactory::createMany(100);
```

Comment:
That's pretty big, 25 seems to be more reasonable

```suggestion
        TagFactory::createMany(25);
```